dev/com.ibm.ws.persistence_fat/bnd.bnd

```diff
@@ -19,6 +19,7 @@ src: \
 	test-applications/consumer/src
 
 fat.project: true
+tested.features: jaxb-3.0, servlet-5.0, jpacontainer-3.0, jpa-3.0, ejblite-4.0
```

Comment:
need to change "ejbLite-4.0" to "enterpriseBeansLite-4.0"

Reply:
addressed!